doc/source/security/encrypting-system-tablespace.rst

```diff
@@ -0,0 +1,61 @@
+.. _encrypting-system-tablespace:
+
+==================================================================
+Encrypting the System Tablespace
+==================================================================
+
+|Percona Server| supports system tablespace encryption. The InnoDB system tablespace is encrypted with the Master key encryption. 
+
+The limitation is the following:
+
+* You cannot convert the system tablespace from the encrypted state to the unencrypted state, or the unencrypted state to the encrypted state. You should create a new instance with the system tablespace in the required state and transfer the user tables to that instance.
+
+To enable system tablespace encryption, edit the my.cnf file with the following:
+
+* Add the :variable:`innodb_sys_tablespace_encrypt`
+* Edit the value to "ON"
+
+System tablespace encryption can only be enabled with the
+``--initialize`` option
+
+You can create an encrypted table as follows:
+
+.. code-block:: MySQL
+
+    mysql> CREATE TABLE table_name TABLESPACE=innodb_system ENCRYPTION='Y';
+
+After you have encrypted the system tablespace, you cannot re-convert the tablespace to unencrypted. A new instance should be created and the user tables should be transferred to the instance.
+
+System Variables
+------------------------------------------------------------------
+
+.. variable:: innodb_sys_tablespace_encrypt
+
+   :cli: ``--innodb-sys-tablespace-encrypt``
+   :dyn: No
+   :scope: Global
+   :vartype: Boolean
+   :default: ``OFF``
+
+Enables the encryption of the InnoDB system tablespace. 
+
+.. seealso::
+
+   |MySQL| Documentation: mysql system Tablespace Encryption
+   https://dev.mysql.com/doc/refman/8.0/en/innodb-tablespace-encryption.html#innodb-mysql-tablespace-encryption-enabling-disabling
```

Comment:
404

Reply:
"This page has moved or been replaced. The new page is located here:"

Reply:
Ack. A new version has been released, pages updated.